exercises/reverse-string/tests/reverse-string.rs

```diff
@@ -62,3 +62,11 @@ fn test_a_palindrome() {
 fn test_wide_characters() {
     process_reverse_case("子猫", "猫子");
 }
+
+#[test]
+#[ignore]
+#[cfg(feature="graphemes")]
```

Comment:
`graphemes` here, but `grapheme` in the README - which is intended?

I cannot run this test with either option currently:

```$ cargo test --features grapheme -- --ignored
   Compiling reverse_string v1.0.0 (file:///home/pt/src/exercism/rust/exercises/reverse-string)
    Finished dev [unoptimized + debuginfo] target(s) in 1.9 secs
     Running target/debug/deps/reverse_string-5106bfa554d6b4ff

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/reverse_string-a3483aefc3b6f53e

running 5 tests
test test_a_palindrome ... ok
test test_a_word ... ok
test test_a_capitalized_word ... ok
test test_a_sentence_with_punctuation ... ok
test test_wide_characters ... ok

test result: ok. 5 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out

   Doc-tests reverse_string

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
```

(Note that the `test_grapheme_clusters` was not run)


```
$ cargo test --features graphemes -- --ignored 
error: Package `reverse_string v1.0.0 (file:///home/pt/src/exercism/rust/exercises/reverse-string)` does not have these features: `graphemes`
```

Reply:
My bad, thanks for spotting.

I've corrected it to be singular `grapheme` throughout